topics/reference/easy/swap-object.tex

```diff
@@ -17,5 +27,7 @@
 \end{lstlisting}
 
 \begin{solution}
-When calling \texttt{swap} with a \texttt{Point} object, we're passing a reference to the original \texttt{Point} object. The object's instance variables can be changed from within \texttt{swap} and will remain changed after we exit from the function.
+When calling \texttt{swap} with a \texttt{Coordinate} object, we're passing a reference to the original \texttt{Coordinate} object. The object's instance variables can be changed from within \texttt{swap} and will remain changed after we exit from the function.
```

Comment:
I like converting `\texttt{...}` to `\lstinline$...$` since it provides syntax highlighting where applicable.